components/ScheduleSection.js

```diff
@@ -0,0 +1,290 @@
+    <div className='column-md-2 column-xs-12 time'>{time}</div>
+    <div className='column-md-5 column-xs-12 main'>{children}</div>
+    <style jsx>{`
+      .row {
+        position: relative;
+        display: flex;
+        flex-wrap: wrap;
+        padding-left:7.5px;
+        padding-right:7.5px;
+      }
+      .column-md-5 {
+        width: 41.65%;
+      }
+      .column-md-2 {
+        width: 16.66%;
+      }
+      .column-md-8 {
+        width: 66.64%;
+      }
+      .column-md-10 {
+        width: 83.33%;
+      }
+      .time, .main {
+        font-size:28px;
+      }
+      ._margin-bottom-30 {
+        margin-bottom: 30px;
+      }
+      @media (max-width: 768px){
+        .column-xs-12 {
+          width: 100%;
+        }
+      }
+    `}</style>
+  </div>
+)
+
+const BreakRow = ({ time }) => (
+  <div className='row _margin-bottom-30'>
+    <div className='column-md-2 column-xs-12 time'>{time}</div>
+    <div className='column-md-8 column-xs-12 container-break'><div className='break' /></div>
+    <style jsx>{`
+      .row {
+        position: relative;
+        display: flex;
+        flex-wrap: wrap;
+        padding-left:7.5px;
+        padding-right:7.5px;
+      }
+      .column-md-5 {
+        width: 41.65%;
+      }
+      .column-md-2 {
+        width: 16.66%;
+      }
+      .column-md-8 {
+        width: 66.64%;
+      }
+      .column-md-10 {
+        width: 83.33%;
+      }
+      .time {
+        font-size:28px;
+      }
+      ._margin-bottom-30 {
+        margin-bottom: 30px;
+      }
+      .break {
+        height: 2px;
+        width: 20%;
+        background-color: #3A3A3A;
+      }
+      .container-break {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+      @media (max-width: 768px){
+        .container-break {
+          margin-top: 15px;
+        }
+        .column-xs-12 {
+          width: 100%;
+        }
+      }
+    `}</style>
+  </div>
+)
+
+const SessionRow = ({ time, children }) => (
+  <div className='row _margin-bottom-30'>
+    <div className='column-md-2 column-xs-12 time'>{time}</div>
+    {children}
+    <style jsx>{`
+      .row {
+        position: relative;
+        display: flex;
+        flex-wrap: wrap;
+        padding-left:7.5px;
+        padding-right:7.5px;
+      }
+      .column-md-5 {
+        width: 41.65%;
+      }
+      .column-md-2 {
+        width: 16.66%;
+      }
+      .column-md-8 {
+        width: 66.64%;
+      }
+      .column-md-10 {
+        width: 83.33%;
+      }
+      .time {
+        font-size:28px;
+      }
+      ._margin-bottom-30 {
+        margin-bottom: 30px;
+      }
+      @media (max-width: 768px){
+        .column-xs-12 {
+          width: 100%;
+        }
+      }
+    `}</style>
+  </div>
+)
+
+const SessionItem = ({ topic, speakerName, speakerAvatar, speakerTitle }) => {
+  return (
+    <div className='column-md-5 column-xs-12'>
+      <div className='row'>
```

Comment:
For SessionItem, I think we didn’t need to any responsive components (such as `.row` or `.column` terminology here), because each speaker item looks the same whether mobile or desktop. A SessionItem might look like this:

```js
const SessionItem = ({ topic, speakerName, speakerAvatar, speakerTitle }) => {
  return (
    <ScheduleColumn size={5}>
      <div className='speaker'>
        <div className='avatar'>
          <img alt='Photo' aria-hidden src={getSpeakerAvatar(speakerName)} />
        </div>
        <div className='info'>
          <div className='topic'>
            {topic}
          </div>
          <div className='speaker'>
            {speakerName} {speakerTitle}
          </div>
        </div>
      </div>
      <style jsx>{`
        /* Layout */
        .speaker {
          display: flex;
          align-items: center;
        }
        .avatar { flex: none; }
        .info { flex: auto; }

        /* Styling */
        .avatar img {
          /* border-radius stuff here */
        }
        .info {
          text-align: left;
        }
        .topic {
          color: #00E0FF;
          font-weight: bold;
        }
        @media (max-width: 768px){
          .topic, .speaker {
            font-size: 18px;
          }
        }
      `}</style>
    </div>
  )
}
```